tests/test_bookmarks.py

```diff
@@ -127,4 +144,4 @@ def test_run(self):
                     self.assertIsNone(second_bookmark_key_value)
 
                     # Verify the number of records in the second sync is the same as the first
-                    self.assertEqual(second_sync_count, first_sync_count)
\ No newline at end of file
+                    self.assertEqual(second_sync_count, first_sync_count)
```

Comment:
Add the test to `Verify at least 1 record was replicated in the second sync`

Reply:
Done